profile/profile.go

```diff
@@ -715,6 +715,36 @@ func (s *Sample) HasLabel(key, value string) bool {
 	return false
 }
 
+// SetNumLabel sets the specified key to the specified value for all samples in the
+// profile.
+func (p *Profile) SetNumLabel(key string, value []int64) {
+	for _, sample := range p.Sample {
+		if sample.NumLabel == nil {
+			sample.NumLabel = map[string][]int64{key: value}
+		} else {
+			sample.NumLabel[key] = value
+		}
+	}
+}
+
+// RemoveNumLabel removes all labels associated with the specified key for all
+// samples in the profile.
+func (p *Profile) RemoveNumLabel(key string) {
+	for _, sample := range p.Sample {
+		delete(sample.NumLabel, key)
+	}
+}
+
+// HasNumLabel returns true if a sample has a label with indicated key and value.
+func (s *Sample) HasNumLabel(key string, value int64) bool {
```

Comment:
I wonder if this helper is added for symmetry or there is a use case for it. I can see that for string labels checking the presence of a specific value of a specific label can be useful since string labels are basically enums and enum values are pets. For numeric labels values are more of cattle IMO and checking for a specific value seems exotic.

Reply:
I don't have a specific use case for it, and I'm fine with taking it out